src/plugins/simulator/visualizations/qt-opengl/qtopengl_camera.cpp

```diff
@@ -73,7 +73,31 @@ namespace argos {
 
    CQTOpenGLCamera::CQTOpenGLCamera() :
       m_bEnableTimeline(false),
+      m_bHasTimeline(false),
       m_unLoop(0) {
+      /* automatically place the cameras at 30 degree increments
+         around the arena */
+      const CVector3& cArenaSize = 
+         CSimulator::GetInstance().GetSpace().GetArenaSize();
+      const CVector3& cArenaCenter = 
+         CSimulator::GetInstance().GetSpace().GetArenaCenter(); 
+      for(UInt32 un_index = 0; 
+            un_index < m_arrPlacements.size();
+            ++un_index) {
+         CRadians cAngle(CRadians::PI_OVER_SIX);
+         cAngle *= static_cast<Real>(un_index);
+         m_arrPlacements[un_index].Position = 
+            CVector3::X * cArenaSize.Length();
+         m_arrPlacements[un_index].Position.RotateZ(cAngle);
+         m_arrPlacements[un_index].Position += cArenaCenter;
+         m_arrPlacements[un_index].Target = cArenaCenter;
+         m_arrPlacements[un_index].Target.SetZ(0);
+         m_arrPlacements[un_index].Up = 
+            /* position - target? */
+            CVector3(0.0, 0.0, 1.0).Normalize();
```

Comment:
Use CVector3::Z for readability (I think that exists?)

Reply:
Also, why would you normalize a vector that is created already normalized?

Reply:
Fixed in 1588d020fc01cde3cc3ef7750bbbb8b200b85b44